LykkeApi2/Modules/ClientsModule.cs

```diff
@@ -28,8 +28,8 @@ protected override void Load(ContainerBuilder builder)
                 .As<IClientAccountService>()
                 .WithParameter("baseUri", new Uri(_serviceSettings.CurrentValue.ClientAccountServiceUrl));
 
-            builder.RegisterType<ClientSettingsClient>()
-                .As<IClientSettingsClient>();
+            builder.RegisterLykkeServiceClient(_serviceSettings.CurrentValue.ClientAccountServiceUrl);
+            builder.Register(c => c.Resolve<IClientAccountClient>()).As<IClientAccountSettingsClient>().SingleInstance();   // this should be implemented in Lykke.Service.ClientAccount.Client
```

Comment:
Added in version 53 of the package, please update.

Reply:
Ok.